ui/src/components/CustomField.tsx

```diff
@@ -42,82 +43,109 @@ export const CustomFieldName = ({
       name={name}
       defaultValue={props.defaultValue}
       style={{ width: "100%" }}
+      asChild
     >
-      <Flex
-        gap="medium"
-        style={{
-          alignItems: "baseline",
-          justifyContent: "space-between",
-        }}
-      >
-        <FormLabel>
-          <Text>{inputTitle}</Text>
-        </FormLabel>
-        <FormMessage match="valueMissing">Please enter your {name}</FormMessage>
-        <FormMessage match="typeMismatch">
-          Please provide a valid {title}
-        </FormMessage>
-      </Flex>
-      <FormControl asChild>
-        <Controller
-          defaultValue={props.defaultValue}
-          name={name}
-          control={control}
-          render={({ field }) => {
-            switch (variant) {
-              case "textarea": {
-                return (
-                  <textarea
-                    {...field}
-                    placeholder={
-                      placeholder ||
-                      `Enter your ${title?.toLowerCase() || name}`
-                    }
-                    style={style}
-                  />
-                );
-              }
-              case "select": {
-                const { ref, onChange, ...rest } = field;
-                return (
-                  <Select
-                    {...rest}
-                    onValueChange={(value: any) => field.onChange(value)}
-                  >
-                    <Select.Trigger
-                      ref={ref}
-                      style={{ height: "26px", width: "100%" }}
+      <Flex direction="column" gap="small">
+        <Flex
+          gap="medium"
+          style={{
+            alignItems: "baseline",
+            justifyContent: "space-between",
+          }}
+        >
+          <FormLabel>
+            <Text>{inputTitle}</Text>
+          </FormLabel>
+          <FormMessage match="valueMissing">
+            Please enter your {name}
+          </FormMessage>
+          <FormMessage match="typeMismatch">
+            Please provide a valid {title}
```

Comment:
`title` is marked as optional in `CustomFieldNameProps`. Should we have a fallback value here or default value in the args?

Reply:
yes, we should do it. lets create a task for that